### PR TITLE
add openam() check (#205)

### PR DESCRIPTION
**Pull request by vetinari**
Check usage:
        openam().auth(chain="SomeChain", realm="/employees")

Username and password for the authentication are taken from
ZMONs (docker) environment.

### CODE REVIEW

**Comment by vetinari** [comment]
:+1:

File: zmon_worker_extras/check_plugins/openam.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            timeout=10,
+            max_retries=0,
+            verify=True,
+            headers=None,
+            ):
+        self.url = url
+        self.params = params or {}
+        self.headers = headers or {}
+
+    def auth(self, chain=None, realm=None, user=os.getenv("OPENAM_USER"), password=os.getenv("OPENAM_PASSWORD")):
```
Comment (mohabusama) [inline zmon_worker_extras/check_plugins/openam.py]: Isn't it possible to set ``OPENAM_USER`` and ``OPENAM_PASSWORD`` in plugin config similar to ``url``? Just to be consistent with the plugin system.

File: zmon_worker_extras/check_plugins/openam.worker_plugin [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[Core]
+Name = openam
+Module = openam
+
+[Documentation]
+Author = John Doe
```
Comment (lmineiro) [inline zmon_worker_extras/check_plugins/openam.worker_plugin]: John Doe? :)

File: zmon_worker_extras/check_plugins/openam.py (hunk 1 above)
Comment (vetinari) [inline zmon_worker_extras/check_plugins/openam.py]: how do we get the values only for our instance(s)?

**Comment by mohabusama** [comment]
Looks good. Only missing some ``flake8`` errors to pass the build:
https://travis-ci.org/zalando-zmon/zmon-worker/builds/203749158#L2874

**Comment by mohabusama** [comment]
Great, Thanks!

**Comment by mohabusama** [comment]
👍

**Pull request merged**